pennylane/__init__.py

```diff
@@ -338,3 +307,45 @@ def run_cnot():
 def version():
     """Returns the PennyLane version number."""
     return __version__
+
+
+# add everything as long as it's not a module and not prefixed with _
+_all = sorted(
```

Comment:
How come we're sorting here?

Reply:
Good question @antalszava.. I do not remember why I am sorting this.